models/cam-se/model_mod.f90

```diff
@@ -593,6 +600,11 @@ subroutine model_interpolate(state_handle, ens_size, location, obs_qty, interp_v
 !SENote Do further study of how we want to return istatus for various failures
 ! For now return istatus 12 for any of the failure modes
 if (any(istatus /= 0)) then
+   ! Debug vert info
+   if(debug_level > 12) then
+      write(string1,*)'get_se_quad_vals failed istatus(:) = ', istatus
+      call error_handler(E_MSG,routine,string1,source,revision,revdate)
+   endif
```

Comment:
whats the goal of adding this message? And does it need to be in this pull request?
E_MSG means only task 0 will print this. If other tasks have a non-zero istatus you won't get a message.